java/maven.junit.ui/src/org/netbeans/modules/maven/junit/ui/MavenJUnitNodeOpener.java

```diff
@@ -188,14 +188,18 @@ public void openCallstackFrame(Node node, @NonNull String frameInfo) {
         if (testfo != null && file == null && methodNode.getTestcase().getTrouble() != null && lineNumStorage[0] == -1) {
                 //213935 we could not recognize the stack trace line and map it to known file
             //if it's a failure text, grab the testcase's own line from the stack.
+            String fqMethodName= methodNode.getTestcase().getClassName()+ '.'+ methodNode.getTestcase().getName();
```

Comment:
This does not work (at least in my tests it does not). The name of the testcase  also holds the class name, but maybe that is already the problem and we should fix it at the root?!

This is the code I use to fix the nested test cases:

https://github.com/apache/netbeans/pull/8664/files#diff-20f21830cc72d9977dabeae930571d61083d0f61de40c3fd9637e5c9c69ec461R223-R239

Seems for ant this is not a problem as the report is extracted differently.